File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/IQuestMediatorFacade.java
package ru.nekit.android.qls.quest.view;

import ru.nekit.android.qls.quest.view.mediator.IQuestMediator;

public interface IQuestMediatorFacade extends IQuestMediator {

    IQuestMediator getTitleMediator();

    IQuestMediator getContentMediator();

    IQuestMediator getAlternativeAnswerMediator();

}

File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/IQuestMediatorFacade.kt
package ru.nekit.android.qls.quest.view

import ru.nekit.android.qls.quest.view.mediator.IQuestMediator

//ver 1.0
interface IQuestMediatorFacade : IQuestMediator {

    val titleMediator: IQuestMediator

    val contentMediator: IQuestMediator

    val answerMediator: IQuestMediator

}